src/commonMain/kotlin/fr/acinq/lightning/payment/OutgoingPaymentHandler.kt

```diff
@@ -46,12 +46,16 @@ class OutgoingPaymentHandler(val nodeId: PublicKey, val walletParams: WalletPara
 
     private fun getPaymentAttempt(childId: UUID): PaymentAttempt? = childToParentId[childId]?.let { pending[it] }
 
-    suspend fun sendPayment(request: SendPayment, channels: Map<ByteVector32, ChannelState>, currentBlockHeight: Int): SendPaymentResult {
+    suspend fun sendPayment(request: SendPayment, channels: Map<ByteVector32, ChannelState>, features: Features, currentBlockHeight: Int): SendPaymentResult {
```

Comment:
Why didn't you use `request.paymentRequest.features`?

Reply:
Those are the node's features, not the invoice's.
I renamed the field in https://github.com/ACINQ/lightning-kmp/pull/367/commits/49928390d38c8cee62abb458fae2e73a1ce5b9ea

Reply:
We could almost add those to the `OutgoingPaymentHandler` constructor, but it's fine that way.

Reply:
Let's see later if we need it elsewhere than in the `sendPayment` function, we can move it to the `OutgoingPaymentHandler` at that point.

Reply:
Hu, just realize that I actually added `nodeParams` to the constructor in #355. What are the odds right?

Reply:
Done in f818299c1829bfc57a63b20fab83aea6d3542606.